pkg/builddocker/build.go

```diff
@@ -45,8 +46,78 @@ func GenerateDockerfile(w io.Writer, env hcl2nix.OCIArtifact, platform string) e
 	}
 
 	return nil
+
+}
+
+// ModifyDockerfile modifies the Dockerfile with the specified tag
+func ModifyDockerfile(file *os.File, dev bool, tag string) ([]string, error) {
+	lines, err := readDockerFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	reslines, err := editDockerfile(lines, dev, tag)
+	if err != nil {
+		return nil, err
+	}
+
+	return reslines, nil
+}
+
+func readDockerFile(file *os.File) ([]string, error) {
+	scanner := bufio.NewScanner(file)
+	lines := []string{}
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	if err := scanner.Err(); err != nil {
+		return nil, fmt.Errorf("error reading Dockerfile: %v", err)
+	}
+	return lines, nil
 }
 
+	func editDockerfile(lines []string, dev bool, tag string) ([]string, error) {
+		var searchTag string
+		if dev {
+			searchTag = "# bsfimage:dev"
```

Comment:
Shall we do it with/without space? I am sure people will do either and if we look for exact string match, we'll fail. 

Or maybe the searchTag should just be `bsfimage:dev`.

Reply:
> Or maybe the searchTag should just be bsfimage:dev.
yup even I was thinking about this, doing it